src/driver/src/DefaultDriver.cpp

```diff
@@ -116,8 +116,14 @@ DefaultDriver::OutcomeCode doRunLogic(const DefaultDriver::ProgramOptions& optio
         return DefaultDriver::OutcomeCode::kSuccess;
     }
 
+    NodeSource nodeSource{YAML::Dump(yaml),
```

Comment:
I really want to avoid exposing anything about yaml-cpp itself within `genny::Node`. I think there's a good case to be made for creating a helper thing in `testlib` that does this dance but from a purist's perspective `genny::Node` doesn't expose the fact that it uses `yaml-cpp` under the covers.